.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "packages-for-isopackets/blaze"]
-	path = packages-for-isopackets/blaze
-	url = https://github.com/meteor/blaze.git
+[submodule "packages/non-core/blaze"]
+    path = packages/non-core/blaze
```

Comment:
@tmeasday What do you think of reusing the `packages/non-core/` directory for independent submodules like `blaze`?